server/sockets/cohort.js

```diff
@@ -138,10 +138,42 @@ const updateCohortHeaderStatus = socket => {
   });
 };
 
+const removeCohortMember = (socket, allCohortsClients, cohortClients) =>
+  socket.on(CohortActionTypes.REMOVE_MEMBER_SUCCESS, data => {
+    const { cohortId, userId } = data;
+
+    socket.broadcast
+      .to(`cohort-${cohortId}`)
```

Comment:
Please extract helper functions to reduce duplication and possibility or errors:

```javascript
const cohortChannel = cohortId => `cohort-${cohortId}`;

...

const { cohortId, userId } = data;
const channel = cohortChannel(cohortId);

socket
  .broadcast
  .to(channel)
  .emit(CohortActionTypes.REMOVE_MEMBER_SUCCESS, data);
```